pgml-admin/app/static/css/base.css

```diff
@@ -57,6 +57,7 @@ footer, header, hgroup, menu, nav, section {
 }
 body {
 	line-height: 1;
+    height: 100vh;
```

Comment:
I don't know if this is safe, how does it work with overflow?

Reply:
Right, good call, will replace with `min-height`